packages/lib/formulas/flatten/formula.json

```diff
@@ -2,16 +2,28 @@
   "$schema": "../../schemas/libFormula.schema.json",
   "name": "Flatten",
   "description": "Flatten a nested Array.",
+<<<<<<< HEAD
+=======
```

Comment:
You've got a few issues where the changes didn't merge correctly!

In your code editor, do a search for <<<< to find them all and fix them up.